examples/with-passport/lib/redirect.js

```diff
@@ -0,0 +1,10 @@
+import Router from 'next/router'
+
+export default function redirect (res, path) {
+  if (typeof window === 'undefined') {
+    res.writeHead(302, { Location: path })
+    res.end()
+  } else {
+    Router.push(path)
```

Comment:
Router.push('error', path) will immediately clear everything, including redux and History, so the user can not go back with the back button. do you agree?